.github/workflows/tpu_ci.yml

```diff
@@ -30,6 +32,7 @@ jobs:
         env:
           PJRT_DEVICE: TPU
         run: |
+          pip install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cu121 --user
```

Comment:
I know the error originates from the other file, but we don't need `torchaudio` and there is a CPU index https://download.pytorch.org/whl/nightly/cpu

We don't need to install the CUDA versions on TPU

Reply:
fixed.